Exec/radiation_tests/RadBreakout/filt_prim.H

```diff
@@ -0,0 +1,117 @@
+#ifndef filt_prim_H
+#define filt_prim_H
+
+#include <AMReX.H>
+#include <filter.H>
+
+AMREX_GPU_HOST_DEVICE AMREX_INLINE
+void filt_prim (int i, int j, int k,
+                Array4<Real> const& Stmp,
+                Array4<Real> const& Snew,
+                int filt_T, int S,
+                const GeometryData& geomdata,
+                Real time)
+{
+    using namespace filter;
+
+    bool filtered = false;
+
```

Comment:
I think that we should have an assert here that this filtering is only implemented for 1-d

Reply:
I'll add an assert, but the change for dealing with multiple momentum components should come in some other PR since that's actually a change to the problem setup (and it wouldn't just be the kinetic energy, it would also be an actual filtering on the other momentum components too).